internal/batches/docker/image.go

```diff
@@ -103,7 +103,7 @@ func (image *image) Ensure(ctx context.Context) error {
 			// docker image inspect will return a non-zero exit code if the image and
 			// tag don't exist locally, regardless of the format.
 			var digest string
-			if digest, err = inspectDigest(); errors.HasType(err, &fastCommandTimeoutError{}) {
+			if digest, err = inspectDigest(); errors.HasType[*fastCommandTimeoutError](err) {
```

Comment:
`errors.HasType` was changed in Sourcegraph so I had to update the call here. (See https://github.com/sourcegraph/sourcegraph/pull/63024)